MobiFlight/InputConfig/ButtonInputConfig.cs

```diff
@@ -10,13 +8,15 @@ public class ButtonInputConfig : IXmlSerializable, ICloneable
     {
         public InputAction onPress;
         public InputAction onRelease;
+        public InputAction onLongRelease;
 
 
         public object Clone()
         {
             ButtonInputConfig clone = new ButtonInputConfig();
             if (onPress != null) clone.onPress = (InputAction)onPress.Clone();
             if (onRelease != null) clone.onRelease = (InputAction)onRelease.Clone();
+            if (onLongRelease != null) clone.onLongRelease = (InputAction)onLongRelease.Clone();

```

Comment:
Great you though about the unit tests.

https://github.com/MobiFlight/MobiFlight-Connector/blob/0944c0a61e394253a74654bda17383b30bc1e231/MobiFlightUnitTests/MobiFlight/InputConfig/ButtonInputConfigTests.cs#L27

We have to extend them for this new property accordingly.

Reply:
Done